 <launch>
 <!--  <node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 map base_link 10" /> -->
  <node name="map_server" pkg="map_server" type="map_server" args="/home/kush/catkin_ws/src/robot_navigation/launch/map_pnf.yaml" />
  <node name="robot_navigation" pkg="robot_navigation" type="robot_navigation_" output="screen"> 
     <rosparam file="$(find robot_navigation)/launch/costmap_common_params.yaml" command="load" ns="global_costmap"/>
     <rosparam file="$(find robot_navigation)/launch/costmap_common_params.yaml" command="load" ns="local_costmap"/>
     <rosparam file="$(find robot_navigation)/launch/local_costmap_params.yaml" command="load" />
     <rosparam file="$(find robot_navigation)/launch/global_costmap_params.yaml" command="load" />
     <rosparam file="$(find robot_navigation)/launch/costmap_converter_params.yaml" command="load" />
     <rosparam file="$(find robot_navigation)/launch/teb_local_planner_params.yaml" command="load" />
 </node> 
</launch>
